### ISSUE TITLE
Loki test fails for Accessibility plugin

### ISSUE THREAD

**Issue by bala3nadh-UI** [oblador/loki#153]
I have added "storybook-addon-a11y" plugin for React-storybook to display the Accessibility results for each of the components -
<img width="1675" alt="Screen Shot 2019-06-10 at 2 04 12 PM" src="https://user-images.githubusercontent.com/30519317/59188429-ab769800-8b88-11e9-82d8-1919ff6da1a6.png">

**Issue: Loki tests are failing with some white space showing on top of the screenshots.**
Screenshot after the plugin added -
<img width="722" alt="Screen Shot 2019-06-10 at 2 05 44 PM" src="https://user-images.githubusercontent.com/30519317/59188511-deb92700-8b88-11e9-82f9-e8dff8352939.png">

Screenshot before the plugin added -
<img width="714" alt="Screen Shot 2019-06-10 at 2 06 23 PM" src="https://user-images.githubusercontent.com/30519317/59188565-f4c6e780-8b88-11e9-834d-b43340bde0e1.png">

More details:
Loki selector used -
	"loki": {
		"chromeSelector": ".uil-story-wrapper > *",
         }

**Comment by vvscode** [comment]
@oblador  I've checked storybook from build directory - there is no such whitespace above

**Comment by JohnAlbin** [comment]
Yeah, I'm using the a11y plugin and I do NOT experience that whitespace issue with Loki either.

**Comment by oblador** [comment]
Closing as invalid. Please reopen if you have a reliable way to reproduce and you think it's rooted in loki and not the plugin itself.
